lib/we/call/version.rb

```diff
@@ -1,5 +1,5 @@
 module We
   module Call
-    VERSION = "0.8.0"
+    VERSION = "0.9.a"
```

Comment:
This gem is following SemVer, which suggests a format more like `0.9.0-alpha1` https://semver.org/